content/en/docs/contribute/style/style-guide.md

```diff
@@ -113,7 +115,7 @@ The copy is called a "fork". | The copy is called a "fork."
 
 ## Inline code formatting
 
-### Use code style for inline code, commands, and API objects
+### Use code style for inline code, commands, and API objects {#code-style-inline-code-commands-api-objects}
```

Comment:
I anchor text is wordy. The table caption for this section needs updating.

Reply:
I fixed the anchor text, and searched for references to the anchor. 

BTW, do you have a suggested tool to search for broken #links in a local repo?

I'm not sure what you mean by the table caption needing updating. I think this is the only specific guidance, https://v1-17.docs.kubernetes.io/docs/contribute/style/style-guide/#tables

I attempted to conform the table captions to the prevailing style in the document.  

Thanks again!